src/components/Dashboard.js

```diff
@@ -48,6 +49,11 @@ export const Dashboard = () => {
     setToCurrency(temporaryFromCurrencyKeeper)
   }
 
+  const setInitialCurrencies = (apiRates) => {
+    setFromCurrency(findAndReturnCurrencyByCode(apiRates, initialSelectValueUSD))
```

Comment:
What do you think about changing these variables names to: `initialSelectFromValue` and `initialSelectToValue`? 
Because in this case, if we change the default values of fields, we will need to add new const and change it in two places.

Reply:
That's a good point. Changed